modules/twinklexfd.js

```diff
@@ -740,6 +809,13 @@ Twinkle.xfd.callbacks = {
 			}
 			usertalkpage.setFollowRedirect(true);
 			usertalkpage.append();
+		},
+		delsortListing: function(pageobj) {
+			var discussionPage = pageobj.getCallbackParameters().discussionPage;
+			var text = pageobj.getPageText().replace("directly below this line -->", "directly below this line -->\n{{" + discussionPage + "}}");
+			pageobj.setPageText(text);
+			pageobj.setEditSummary('Listing [[:' + discussionPage +']].' + Twinkle.getPref('summaryAd'));
+			pageobj.save();
```

Comment:
```suggestion
			pageobj.setCreateOption('nocreate');
			pageobj.save();
```

Just to be safe, eh?